raft.c

```diff
@@ -1367,6 +1404,8 @@ void RaftReqFree(RaftReq *req)
 
         RedisModule_UnblockClient(req->client, NULL);
     }
+
+    RedisModule_LatencyAddSample(RaftReqTypeStr[req->type],  (mstime_t) ((timeMonotonicNanos() - req->begin) / 1000000));
```

Comment:
RedisModule_LatencyAddSample() does not seem thread-safe to me. So, I believe we should guarantee that we call it only in raft thread. Seems like we can call RaftReqFree() in redis thread itself? Hope I'm all wrong :)